gq_filter_wdl/BenchmarkGqFilter.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Can delete these now that you have templates?

Reply:
Yeah, that was there accidentally. Deleted.